sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1,5 +1,58 @@
 namespace Azure.AI.TextAnalytics
 {
+    public partial class AbstractSummaryAction
+    {
+        public AbstractSummaryAction() { }
+        public AbstractSummaryAction(Azure.AI.TextAnalytics.AbstractSummaryOptions options) { }
+        public string ActionName { get { throw null; } set { } }
+        public bool? DisableServiceLogs { get { throw null; } set { } }
+        public int? MaxSentenceCount { get { throw null; } set { } }
+        public string ModelVersion { get { throw null; } set { } }
+    }
+    public partial class AbstractSummaryActionResult : Azure.AI.TextAnalytics.TextAnalyticsActionResult
+    {
+        internal AbstractSummaryActionResult() { }
+        public Azure.AI.TextAnalytics.AbstractSummaryResultCollection DocumentsResults { get { throw null; } }
+    }
+    public partial class AbstractSummaryOperation : Azure.PageableOperation<Azure.AI.TextAnalytics.AbstractSummaryResultCollection>
+    {
+        protected AbstractSummaryOperation() { }
+        public AbstractSummaryOperation(string operationId, Azure.AI.TextAnalytics.TextAnalyticsClient client) { }
+        public virtual System.DateTimeOffset CreatedOn { get { throw null; } }
+        public virtual string DisplayName { get { throw null; } }
+        public virtual System.DateTimeOffset? ExpiresOn { get { throw null; } }
+        public override bool HasCompleted { get { throw null; } }
+        public override bool HasValue { get { throw null; } }
+        public override string Id { get { throw null; } }
+        public virtual System.DateTimeOffset LastModified { get { throw null; } }
```

Comment:
Should it have an "On" postfix similar to the other dates part of this object? Or is that the name used in other actions too?

Reply:
As I know, .NET always use the LastModified in TA.

Reply:
@mssfang is correct. We use `LastModified` all across the Text Analytics .NET library.